src/components/menu/API.tsx

```diff
@@ -0,0 +1,128 @@
+                                <dt>Request Parameters</dt>
+                                <dd>id</dd>
+                                <dt>Example Request</dt>
+                                <dd>https://aoe2cm.net/api/preset/simple</dd>
+                                <dd>
+                                <ul>
+                                    <li>TECHNICAL = 0</li>
+                                    <li>AOK = 1</li>
+                                    <li>CONQUERORS = 2</li>
+                                    <li>FORGOTTEN = 3</li>
+                                    <li>AFRICAN_KINGDOMS = 4</li>
+                                    <li>RISE_OF_RAJAS = 5</li>
+                                    <li>DEFINITIVE_EDITION = 6</li>
+                                    <li>LORDS_OF_THE_WEST = 7</li>
+                                    </ul>
+                                </dd>
+                            </dl>
+
+                            <h5>Add Preset</h5>
+                            <p>Request the selected preset.</p>
+                            <dl>
+                                <dt>Endpoint</dt>
+                                <dd>/api/preset</dd>   
+                                <dt>Request Parameters</dt>
+                                <dd>No Clue</dd>
+                                <dt>Example Request</dt>
+                                <dd>https://aoe2cm.net/api/preset/new</dd>
+                            </dl>
+
+                            <h5>List Preset</h5>
+                            <p>Lists all presets with name and id.</p>
+                            <dl>
+                                <dt>Endpoint</dt>
+                                <dd>/api/preset/list</dd>   
+                                <dt>Request Parameters</dt>
+                                <dd>No parameters</dd>
+                                <dt>Example Request</dt>
+                                <dd>https://aoe2cm.net/api/preset/list</dd>
+                            </dl>
+
+                            <h4>Draft</h4>
+                            <h5>Draft</h5>
+                            <p>Request the selected Draft.</p>
+                            <dl>
+                                <dt>Endpoint</dt>
+                                <dd>/api/draft</dd>  
+                                <dt>Request Parameters</dt>
+                                <dd>name</dd>
+                                <dt>Example Request</dt>
+                                <dd>https://aoe2cm.net/api/draft/FQoju</dd>
+                            </dl>
+                            <h5>Add draft</h5>
+                            <p>Request the selected preset.</p>
+                            <dl>
+                                <dt>Endpoint</dt>
+                                <dd>/api/preset</dd>   
+                                <dt>Request Parameters</dt>
+                                <dd>No Clue</dd>
```

Comment:
Basically, you send a JSON string in the following format:
```json
{
  "preset": {
    "name": string,
    "encodedCivilisations": string,
    "turns": Turn[]
  }
}
```
and get back `{"status": "ok", "draftId": "abcdef"}`